src/test/java/tests/RepositorySelectorPopulationTests.java

```diff
@@ -47,16 +49,17 @@ public void noJsonFiles() {
     public void oneInvalidJsonFile() throws IOException {
         File invalidJson = new File("store/test/dummy-dummy.json");
         assert invalidJson.createNewFile();
-        RepoIO testIO = new RepoIO(true, true);
+        RepoIO testIO = TestController.createTestingRepoIO(Optional.of(new JSONStoreStub()));
```

Comment:
Why don't this one and the above use `Optional.empty()` instead? Is it because there is no JSON Store method being called?